packages/core/docs/reference/migrate/2.5.12/overview.md

```diff
@@ -0,0 +1,40 @@
+# Migrating projects to 2.5.12
+
+## Update `nuxt.config.js`
+
+In this release, we've added the optional  `middlewareUrl` property to define the URL to the Server Middleware. Open the `nuxt.config.js` file and add the `middlewareUrl` property to the `publicRuntimeConfig` object as shown below.
```

Comment:
If it's optional, we need to say when and why you should use it.

Reply:
same here (Joking :P)